src/next.ts

```diff
@@ -1,2 +1,7 @@
-export * as Heading from "./components/Heading/Heading";
-export * as EditableHeading from "./components/EditableHeading/EditableHeading";
+export * from "./components/Heading/Heading";
```

Comment:
I believe it should just be export { default as Heading } from..
Not sure why it exports everything as Heading, are there more stuff being exported from there?

Reply:
types?

Reply:
So they are probably unused

Reply:
why not?

Reply:
I guess no one used them till today. It's only a guess of course.
But if someone did used them then this is a breaking change basically.
But I believe we can make a safe guess no one did (as they would have to use Heading.HeadingProps from what I see?)

Reply:
`dist/esm/next.d.ts` before and after my change looks the same:
```typescript
export { default as Heading } from "../components/Heading/Heading";
export { default as EditableHeading } from "../components/EditableHeading/EditableHeading";
```

So seems like my `export *` in addition to the named export didn't work as I expected as the types exported from this index file